priv/repo/migrations/20210416191647_add_session_stats_index.exs

```diff
@@ -0,0 +1,22 @@
+defmodule Ret.Repo.Migrations.AddSessionStatsIndex do
+  use Ecto.Migration
+
+  @max_year 2030
```

Comment:
2030? Show some faith

Reply:
Hah! That's like 30 decades away in tech years.